performance_manager/lib/gtfs_utils.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+
+import pandas
+
+
```

Comment:
These functions are common to `RT_VEHICLE_POSITIONS` and `RT_TRIP_UPDATES` so moved into a new file